tests/contrib/cassandra/test.py

```diff
@@ -264,7 +264,11 @@ def test_batch_statement(self):
         spans = writer.pop()
         eq_(len(spans), 1)
         s = spans[0]
-        eq_(s.resource, 'BatchStatement')
+        eq_(
+            s.resource,
+            """INSERT INTO test.person_write (name, age, description) VALUES ('Joe', 1, 'a'); """
```

Comment:
I like to put parens around multiline strings, helps to reduce confusion on whether these are separate or joined.

```python

```
eq_(
    s.resource,
    (
         'INSERT INTO test.person_write (name, age, description) VALUES (\'Joe'\, 1, \'a\'); '
         'INSERT INTO test.person_write (name, age, description) VALUES (\'Jane\', 2, \'b\')'
    ),
)